volume/mounts/linux_parser.go

```diff
@@ -100,17 +97,10 @@ func (p *linuxParser) validateMountConfigImpl(mnt *mount.Mount, validateBindSour
 		}
 
 	case mount.TypeVolume:
-		if mnt.BindOptions != nil {
-			return &errMountConfig{mnt, errExtraField("BindOptions")}
-		}
-
```

Comment:
Curious; was there a reason to move these out of the validation? Was this to make it based on API version (did something break if we didn't?)